src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/Startup.cs

```diff
@@ -92,12 +92,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseAuthentication();
```

Comment:
This should have `UseRouting()` above it, and `UseAuthorization()` after it.

Reply:
With endpoint routing, the auth is handled by middleware now, so the authN/AuthZ middleware sit in between `UseRouting` and `UseEndpoints`